### ISSUE TITLE
Increase the size of the covers.

### ISSUE THREAD

**Issue by yeraycito** [janeczku/calibre-web#2118]
In my opinion the book covers on the Calibre-web homepage are too small. On one particular page of a book the cover image seems to me to be a good size. Just as you can choose the number of books per page, you could also choose to increase the size of the covers. Thank you.

**Comment by AutoGitr** [comment]
My2c. 
The covers on the homepage are already reasonably sized, but an option to change the size would still be nice. However when i click on a book and see the cover in the book details view i find most of them to be too small. In comparison to the homepage size most of them are the same size if not slightly larger in the book details view. When i first started Calibre-Web this was the first thing i thought should be improved/ implemented. I have spent a lot of time making sure my e-books have high quality covers but in its current form Calibre-Web isn't doing most of them justice when in the book details. 

A setting or settings controlling book cover size globally and/ or in the different views would be a very nice addition. Especially considering one of the main features of Calibre-Web is its looks.

**Comment by yeraycito** [comment]
I fully agree.

**Comment by fivestones** [comment]
I agree too. It’s be awesome to even use covers as background art zoomed to cover the whole background when viewing a particular book detail page, jellyfin or Plex style.

**Comment by fivestones** [comment]
@OzzieIsaacs I know the ability you are referring to where you can click on a cover and zoom it to fullscreen. This is great. However, what I was talking about in my earlier comment is something different. In some media managers (usually for movies/tv, such as plex and jellyfin), there are options for certain skins where when you are viewing a particular movie, the background of the page is a very enlarged full-screen artwork from the movie, with a transparency over it so it is muted. Here's an example: https://jellyfin.org/images/screenshots/movie_full.png. I just think it would look amazing if you did the same thing in calibre-web, blowing up the book cover to a size big enough that it filled the background of the calibre-web page left to right, cut off the bottom of the image or the top and bottom, and then put a transparency overlay over that so it's muted.
